src/env.cc

```diff
@@ -769,6 +778,8 @@ void Environment::InitializeMainContext(Local<Context> context,
                                         const EnvSerializeInfo* env_info) {
   context_.Reset(context->GetIsolate(), context);
   AssignToContext(context, ContextInfo(""));
+  context->SetAlignedPointerInEmbedderData(
```

Comment:
Do we still need this? It seems like `ContextEmbedderIndex::kContextifyContext` should already be initialized to `nullptr` in `AssignToContext`.

Reply:
Yeah it should no longer be necessary after AssignToContext initializes too. I've removed it.